src/test/regress/specs/isolation_ddl_vs_all.spec

```diff
@@ -0,0 +1,99 @@
+#
+# How we organize this isolation test spec, is explained at README.md file in this directory.
+#
+
+# create range distributed table to test behavior of DDL in concurrent operations
+setup
+{
+	SELECT citus.replace_isolation_tester_func();
```

Comment:
This is done only for few tests. Do we need to do those for other isolation tests too ?